golang/05-empty-volume/volume.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path"
+)
+
+var mounted = func(file string) string { return path.Join("mnt", file) }
+
+type hello struct {
+	Message string `json:"message"`
+}
+
+func main() {
+	var err error
+	// read-write files
```

Comment:
you are missing a os.Mkdir somewhere around here unless the dirs are being created by the mount?

Reply:
dirs are created by the mount, which might be an issue since users will have to prefix the expected mount path to their file related functions, or am i getting this wrong?

Reply:
got it; relative mounts are fine as long as the user is aware; i just retried the example && that works; thanks!